app/src/main/res/layout-land/submitted_answer_item.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
+    <import type="android.view.Gravity" />
+
+    <import type="android.view.View" />
+
+    <variable
+      name="submittedAnswer"
+      type="CharSequence" />
+
+    <variable
+      name="submittedListAnswer"
+      type="org.oppia.app.model.ListOfSetsOfHtmlStrings" />
+
+    <variable
+      name="viewModel"
+      type="org.oppia.app.player.state.itemviewmodel.SubmittedAnswerViewModel" />
+  </data>
+
+  <LinearLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:gravity="@{viewModel.hasConversationView ? Gravity.END : Gravity.START}"
+    app:explorationSplitViewPaddingApplicable="@{viewModel.hasConversationView &amp;&amp; viewModel.isSplitView}"
+    app:explorationSplitViewPaddingBottom="@{@dimen/padding_0}"
```

Comment:
Actually this implementation is correct because here we are bounded to use `match_parent` as width of the parent-layout and because of which we cannot introduced correct margins/paddings. So to compensate of this, the margins mentioned in your comment act as `padding` for `LinearLayout` and the padding mentioned in the comment act as padding for `TextView` which ultimately leads to correct UI.

Reply:
Although the previous implementation was also correct, still made one change just for added clarification,

`LinearLayout` padding changed to `margin` which results same.